src/components/AddressSearch.js

```diff
@@ -117,89 +117,97 @@ const AddressSearch = (props) => {
     };
 
     return (
-        <GooglePlacesAutocomplete
-            fetchDetails
-            suppressDefaultStyles
-            enablePoweredByContainer={false}
-            onPress={(data, details) => {
-                saveLocationDetails(details);
-
-                // After we select an option, we set displayListViewBorder to false to prevent UI flickering
-                setDisplayListViewBorder(false);
-            }}
-            query={{
-                key: 'AIzaSyC4axhhXtpiS-WozJEsmlL3Kg3kXucbZus',
-                language: props.preferredLocale,
-                types: 'address',
-                components: 'country:us',
-            }}
-            requestUrl={{
-                useOnPlatform: 'web',
-                url: `${CONFIG.EXPENSIFY.EXPENSIFY_URL}api?command=Proxy_GooglePlaces&proxyUrl=`,
-            }}
-            textInputProps={{
-                InputComp: TextInput,
-                ref: (node) => {
-                    if (!props.innerRef) {
-                        return;
-                    }
-
-                    if (_.isFunction(props.innerRef)) {
-                        props.innerRef(node);
-                        return;
-                    }
-
-                    // eslint-disable-next-line no-param-reassign
-                    props.innerRef.current = node;
-                },
-                label: props.label,
-                containerStyles: props.containerStyles,
-                errorText: props.errorText,
-                value: props.value,
-                isFormInput: props.isFormInput,
-                inputID: props.inputID,
-                shouldSaveDraft: props.shouldSaveDraft,
-                onBlur: props.onBlur,
-                autoComplete: 'off',
-                onChangeText: (text) => {
-                    if (skippedFirstOnChangeTextRef.current) {
-                        props.onChange({street: text});
-                    } else {
-                        skippedFirstOnChangeTextRef.current = true;
-                    }
-
-                    // If the text is empty, we set displayListViewBorder to false to prevent UI flickering
-                    if (_.isEmpty(text)) {
+        <ScrollView
```

Comment:
Can we please add a comment to explain why this is here?

```js
/*
 * The GooglePlacesAutocomplete component uses a VirtualizedList internally,
 * and VirtualizedLists cannot be directly nested within other VirtualizedLists of the same orientation.
 * To work around this, we wrap the GooglePlacesAutocomplete component with a horizontal ScrollView
 * that has scrolling disabled and would otherwise not be needed
 */
```

Reply:
Makes sense. I'll update it.